samples/app-anonymous-users/nodejs/api/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "app-anonymous-users",
+  "version": "1.0.0",
+  "description": "This sample shows anonymous users support in meeting apps.",
+  "main": "server/index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node server/index.js",
+    "watch": "nodemon server/index.js",
+    "lint": "eslint .",
+    "server": "npm install && node server/index.js",
+    "manifest": "del \"appPackage\\appPackage.zip\" 2> nul && powershell Compress-Archive appPackage/* appPackage/appPackage.zip"
+  },
+  "keywords": [],
+  "author": "Microsoft",
+  "license": "MIT",
+  "dependencies": {
+    "@azure/msal-node": "^1.6.0",
+    "@microsoft/microsoft-graph-client": "^2.2.1",
+    "axios": "^0.21.4",
+    "body-parser": "^1.19.0",
+    "cors": "^2.8.5",
+    "dotenv": "^8.0.0",
+    "express": "^4.17.1",
+    "html-entities": "^1.3.1",
+    "http-proxy": "^1.18.1",
+    "isomorphic-fetch": "^3.0.0",
```

Comment:
Check if we are using all of these packages otherwise remove

Reply:
Fixed